pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.pinterest</groupId>
     <artifactId>secor</artifactId>
-    <version>0.16-SNAPSHOT</version>
+    <version>0.17-SNAPSHOT</version>
```

Comment:
No need to bump up the version.  Bump the version would require us to do a maven publish, and that's a manual process for us.